### DISCUSSION TITLE
PandAi

### DISCUSSION THREAD

**Discussion by Aakidul** [community/community#107005] [Programming Help]
### Body

I Just Created a AI model named Panda.
Panda model is a (GMLM) MODEL.
You can check using this url 👇
https://github.com/Aakidul/PandaAi

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ZephyrusBlaze** [comment]


**Reply by Aakidul** [reply]
THANKS SIR FOR YOUR FEEDBACK ☺️
